mobile/examples/super_resolution/app/src/test/java/ai/onnxruntime/example/superresolution/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package ai.onnxruntime.example.superresolution
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
```

Comment:
Is it possible to add a 'real' test here or does that not work due to dependencies on the ORT/ort-ext libraries?

Reply:
good question. not sure if updating unit test here is workable.

but I noticed for the bertokenizer sample android app, it also leaves the unit test as default one. but adding it to the instrumental test. (aka named as androidTest).

https://github.com/microsoft/onnxruntime-extensions/blob/d8581da434a333e573cdbc51b9558142203c9c8c/java/src/test/android/app/src/androidTest/java/ai/onnxruntime/extensions/apptest/BertTokenizerTest.java#L50

https://github.com/microsoft/onnxruntime-extensions/blob/d8581da434a333e573cdbc51b9558142203c9c8c/java/src/test/android/app/src/test/java/ai/onnxruntime/extensions/apptest/ExampleUnitTest.java#L12

Reply:
I think adding an instrumented test would be good. eventually it would be good to be able to run some 'real' test that from CI builds.

Reply:
updated with a new instrumented test that tests `loadmodel` and `createOrtSession` calls.